readme.md

```diff
@@ -508,6 +508,7 @@
 - [Textpattern](https://github.com/drmonkeyninja/awesome-textpattern#readme) - Lightweight PHP-based CMS.
 - [Drupal](https://github.com/nirgn975/awesome-drupal#readme) - Extensible PHP-based CMS.
 - [Craft CMS](https://github.com/craftcms/awesome#readme) - Content-first CMS.
+- [Sitecore](https://github.com/MartinMiles/Awesome-Sitecore) - A robust digital marketing system that combines CMS with contextual intelligence and omnichannel automation technologies. It also offers a rich variety of tools out of the box for emailing, analytics, campaigns and marketing automation tools for managing website(s).
```

Comment:
This is too verbose. See the other descriptions for inspiration.

Reply:
> This is too verbose. See the other descriptions for inspiration.

Provided less verbose description.